doc/ref_arch/kubernetes/chapters/chapter05.md

```diff
@@ -120,7 +120,7 @@ Secrets must not be stored in scripts or code but provided dynamically at runtim
 ##  5.13  Trusted Registry
 Ensure that the container registry only accepts container images from trusted sources that have tested and validated the images. Where images are provided by third parties, define and follow a formal process to validate compliance with security requirements. Also ensure that access control is applied to registries requiring unique credentials, to limit who can control the build or add images.
 
- - Network access to the registry must run over TLS or VPN connections
+- Network access to the registry must run over TLS or VPN connections
```

Comment:
Can the 'must' be changed to guidance / principle wording please?

Reply:
Humm, not sure. Can we trust a registry if the connection is optionally secure to it? What others think?

Reply:
I would agree with @CsatariGergely

Reply:
I would agree "in principle" with @CsatariGergely and @xmulligan. My suggestion use the following statement:
"Network access to the registry must run over a secured connections, such as TLS, SSl, or VPN."
Just to  not limit to a particular protocol.
@CsatariGergely, please address this discussion and we will accept this PR.

Reply:
Would the following work?  I don't disagree that it's needed, it's just the language I'm wary of as this isn't part of the specification, it's guidance.
```suggestion
- It is strongly recommended that network access to the registry is secured using TLS or VPN connections to ensure trust.
```